packages/react-reconciler/src/ReactFiberExpirationTime.js

```diff
@@ -32,6 +32,10 @@ export const Never = 1;
 // Idle is slightly higher priority than Never. It must completely finish in
 // order to be consistent.
 export const Idle = 2;
+// Continuous Hydration is a moving priority. It is slightly higher than Idle
+// and is used to increase priority of hover targets. It is increasing with
+// each usage so that last always wins.
+export let ContinuousHydration = 3;
```

Comment:
Exporting this value seems a little weird, since we might schedule work at the value of `ContinuousHydration` at one point in time but then if we were to later compare the scheduled `expirationTime` to `ContinuousHydration` they wouldn't match.

Reply:
Makes sense. I'll avoid exporting it.